packages/playground/website/public/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+	"theme_color": "#ffffff",
+	"background_color": "#ffffff",
+	"display": "standalone",
+	"scope": "/",
+	"start_url": "/",
+	"short_name": "WordPress Playground",
+	"description": "WordPress Playground",
+	"name": "WordPress Playground",
+	"icons": [
+		{
+			"src": "/website-server/logo-192.png",
```

Comment:
This path only works on the local dev server. In production it will be just `/logo-192.png`.

Reply:
I added a separate production manifest with the correct url